improver/cli/weighted_blending.py

```diff
@@ -141,12 +141,11 @@ def process(*cubes: cli.inputcube,
         raise RuntimeError('model_id_attr must be specified for '
                            'model blending')
 
-    plugin = WeightAndBlend(
+    result = WeightAndBlend(
         coordinate, weighting_method,
         weighting_coord=weighting_coord, wts_dict=weighting_config,
-        y0val=y0val, ynval=ynval, cval=cval)
-    result = plugin.process(
-        cubes, cycletime=cycletime, model_id_attr=model_id_attr,
-        spatial_weights=spatial_weights_from_mask, fuzzy_length=fuzzy_length,
-        attributes_dict=attributes_config)
+        y0val=y0val, ynval=ynval, cval=cval)(
+            cubes, cycletime=cycletime, model_id_attr=model_id_attr,
+            spatial_weights=spatial_weights_from_mask,
+            fuzzy_length=fuzzy_length, attributes_dict=attributes_config)
```

Comment:
I don't have strong feelings either way. I was endeavouring to make all the ones I touched consistent in style. The differing indentation levels help show the distinction as things stand, but if the 2nd reviewer agrees that splitting these back up is a good idea, I will do so.

Reply:
I agree with Fiona on this.  I think consistency of calls is a good thing but probably not the important thing in this case - since all the plugins are so different in how they look and function (number of arguments, init / process split, etc), aiming for consistency probably actually makes the code base less clear.  We might be able to aim for that in the future though.